examples/oauth-password-limited/README.md

```
@@ -0,0 +1,36 @@
+Password-limited OAuth example for iRacing that authenticates, restores cached
+credentials when available, and fetches `/data` API assets into local files.
+
+## Usage
+
+1. Copy the environment template:
+
+```bash
+cp .env.template .env
+```
+
+2. Update `.env` with your credentials:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)